client/a8c-for-agencies/sections/migrations/migrations-overview-v2/sections/migrations-faqs/index.tsx

```diff
@@ -0,0 +1,198 @@
+import { useTranslate } from 'i18n-calypso';
+import { useCallback } from 'react';
+import PageSection from 'calypso/a8c-for-agencies/components/page-section';
+import { A4A_REFERRALS_PAYMENT_SETTINGS } from 'calypso/a8c-for-agencies/components/sidebar-menu/lib/constants';
+import FoldableFAQ from 'calypso/components/foldable-faq';
+import { useDispatch } from 'calypso/state';
+import { recordTracksEvent } from 'calypso/state/analytics/actions';
+
+import './style.scss';
+
+export default function MigrationsFAQs() {
+	const translate = useTranslate();
+	const dispatch = useDispatch();
+
+	const onFaqToggle = useCallback(
+		( faqArgs: { id: string; buttonId: string; isExpanded: boolean; height: number } ) => {
+			const { id, buttonId, isExpanded } = faqArgs;
+
+			const removeHash = () => {
+				history.replaceState( '', document.title, location.pathname + location.search );
+			};
+
+			// Add expanded FAQ buttonId to the URL hash
+			const addHash = () => {
+				history.replaceState(
+					'',
+					document.title,
+					location.pathname + location.search + `#${ buttonId }`
+				);
+			};
+
+			if ( isExpanded ) {
+				addHash();
+				dispatch(
+					recordTracksEvent( 'calypso_a4a_migrations_faq_open', {
+						faq_id: id,
+					} )
+				);
+			} else {
+				removeHash();
+				dispatch(
+					recordTracksEvent( 'calypso_a4a_migrations_closed', {
+						faq_id: id,
+					} )
+				);
+			}
+		},
+		[ dispatch ]
+	);
+
+	return (
+		<PageSection
+			heading={ translate( 'Frequently asked questions' ) }
+			description={ translate( 'Curious about the details? We’ve got answers.' ) }
+		>
+			<ul className="migrations-faqs">
+				<li>
+					<FoldableFAQ
```

Comment:
[NITPICK] 

I think it's better to keep all the FAQs in a constant and map them through it to display them. What do you think?

Reply:
Good suggestion. this has been applied 👍